src/runtime_src/core/common/memaccess.cpp

```diff
@@ -0,0 +1,217 @@
+/**
```

Comment:
Being that this file is "new", please beautify it according to the XRT standard.